docs/wallet_API.md

```diff
@@ -137,15 +137,15 @@ The Beacon wallet requires an extra step to set up the network to connect to and
 
 await wallet.requestPermissions({
   network: {
-    type: 'mainnet' | 'granadanet' | 'hangzhounet' | 'custom',
+    type: 'mainnet' | 'granadanet' | 'ithacanet' | 'custom',
```

Comment:
consider replacing granadanet by hangzhounet" as it is more recent
'mainnet' | 'hangzhounet' | 'ithacanet' | 'custom',

Reply:
yes. thanks fixed